### ISSUE TITLE
Can you share your training pig pics data?

### ISSUE THREAD

**Issue by PJovy** [xixiareone/counting-pigs#1]
Can you share your training pig pictures with labels?

**Comment by xixiareone** [comment]
thanks to pay attention to our work
we would share u next week.


 
---Original---

**Comment by PJovy** [comment]
> thanks to pay attention to our work we would share u next week.
> […](#)
> ---Original--- From: "PJovy"<notifications@github.com> Date: Mon, Aug 19, 2019 19:13 PM To: "xixiareone/counting-pigs"<counting-pigs@noreply.github.com>; Cc: "Subscribed"<subscribed@noreply.github.com>; Subject: [xixiareone/counting-pigs] Can you share your training pig pics data? (#1) Can you share your training pig pictures with labels? — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or mute the thread.

Thanks.

**Comment by xixiareone** [comment]
This is the augmented  data set including training set verification set and test set. They are from Online.



Extract links：https://pan.baidu.com/s/1JYhtI5QA39gWI0aqAv-BkQ 

password: 9dms 






------------------ 原始邮件 ------------------
发件人: "PJovy"<notifications@github.com>;
发送时间: 2019年8月25日(星期天) 下午4:21
收件人: "xixiareone/counting-pigs"<counting-pigs@noreply.github.com>;
抄送: "Gin"<xixiareone@foxmail.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [xixiareone/counting-pigs] Can you share your training pig picsdata? (#1)



 
thanks to pay attention to our work we would share u next week.
 …
 ---Original--- From: "PJovy"notifications@github.com Date: Mon, Aug 19, 2019 19:13 PM To: "xixiareone/counting-pigs"counting-pigs@noreply.github.com; Cc: "Subscribed"subscribed@noreply.github.com; Subject: [xixiareone/counting-pigs] Can you share your training pig pics data? (#1) Can you share your training pig pictures with labels? — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or mute the thread.
  
Thanks.

**Comment by jvxiao** [comment]
Can anyone share a new link to the image dataset,? the old one is out of date.

**Comment by xixiareone** [comment]
The link can be used, I tried it and can open it.




------------------ 原始邮件 ------------------
发件人: "jvxiao"<notifications@github.com>;
发送时间: 2019年11月4日(星期一) 上午10:47
收件人：  “ xixiareone / counting-pigs” <counting-pigs@noreply.github.com>;
抄送：  “ Gin” <xixiareone@foxmail.com>;“评论” <comment@noreply.github.com>;
主题：  回复：[xixiareone /计数猪]你可以分享你的训练猪picsdata？（＃1）




任何人都可以共享指向图像数据集的新链接吗？旧的已经过时了。
 
— 
因为评论，您收到此消息。
直接回复此电子邮件，在GitHub上查看，或取消订阅。

**Comment by jvxiao** [comment]
I mean the dataset of pig images, not the model's file. The link you offered directs to the hdf5 files. I just need the&nbsp; images and label.


 
---Original---

**Comment by xixiareone** [comment]
This is the augmented  data set including training set verification set and test set. They are from Online.



Extract links：https://pan.baidu.com/s/1KCUdlredPxZAZ0ry0O66TQ
password: hr5m







------------------ 原始邮件 ------------------
发件人: "jvxiao"<notifications@github.com>;
发送时间: 2019年11月4日(星期一) 中午11:24
收件人: "xixiareone/counting-pigs"<counting-pigs@noreply.github.com>;
抄送: "Gin"<xixiareone@foxmail.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [xixiareone/counting-pigs] Can you share your training pig picsdata? (#1)



I mean the dataset of pig images, not the model's file. The link you offered directs to the hdf5 files. I just need the&nbsp; images and label. 
 
 
   
 ---Original---

**Comment by jvxiao** [comment]
Many thanks!

**Comment by darkzealotjg** [comment]
Can anyone share a new link?? doesn't work.

**Comment by Just5D** [comment]
Can anyone share a new link?? doesn't work. the dataset of pig images，thanks！

**Comment by xixiareone** [comment]
链接: https://pan.baidu.com/s/1fT0AnZUi9CsTgp5cAYt5tw 提取码: 91fs




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Just5D"<notifications@github.com&gt;;
发送时间:&nbsp;2020年1月24日(星期五) 凌晨0:13
收件人:&nbsp;"xixiareone/counting-pigs"<counting-pigs@noreply.github.com&gt;;
抄送:&nbsp;"Gin"<xixiareone@foxmail.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [xixiareone/counting-pigs] Can you share your training pig pics data? (#1)




Can anyone share a new link?? doesn't work. the dataset of pig images，thanks！

**Comment by Just5D** [comment]
非常有用的工作，感谢您的慷慨分享！数据集的链接能重新提供吗？上面的过期了，非常感谢！

**Comment by xixiareone** [comment]
链接: https://pan.baidu.com/s/1zfKJLqL_qgeUsp7fGFtdjg 提取码: cf93




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Just5D"<notifications@github.com&gt;;
发送时间:&nbsp;2020年2月3日(星期一) 晚上10:42
收件人:&nbsp;"xixiareone/counting-pigs"<counting-pigs@noreply.github.com&gt;;
抄送:&nbsp;"Gin"<xixiareone@foxmail.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [xixiareone/counting-pigs] Can you share your training pig pics data? (#1)




非常有用的工作，感谢您的慷慨分享！数据集的链接能重新提供吗？上面的过期了，非常感谢！

**Comment by Just5D** [comment]
收到，非常感谢！

**Comment by DuCaoTien** [comment]
@xixiareone I am very interested in your project. You can share your dataset with me?

**Comment by zmiecer** [comment]
@xixiareone +1, I also want images, dot maps and labels, like in counting/datasets/images/, to try to reproduce your results

**Comment by BJTUJia** [comment]
@xixiareone +2, the above links are all expired.  Can you share your dataset again? Many thanks!

**Comment by deep-practice** [comment]
@xixiareone +3, the above links are all expired. Can you share your dataset again? Many thanks!

**Comment by chuasonglin** [comment]
@xixiareone +4, the above links are all expired. Can you share your dataset again? Many thanks!

**Comment by AprilWinds** [comment]
> e links are all expired. Can you share your dataset again? Many thanks!

can you share pig dataset? tks

**Comment by 2605759123** [comment]
@xixiareone  the above links are all expired. Can you share your dataset again? Many thanks!

**Comment by 2605759123** [comment]
@xixiareone 链接过期了，可以再分享一下吗，谢谢~
